src/AppBundle/Resources/views/themes/app/pagelayout_variables.html.twig

```diff
@@ -24,6 +24,14 @@
     {% endif %}
 {% endif %}
 
+{# CURRENT LOCATION PATH ID ARRAY #}
+{% set current_location_path = [] %}
+{% if app.request.attributes.get('_route') == 'ez_urlalias' and app.request.attributes.has('view') %}
+    {% if location is defined and location|default(null) is not empty %}
+        {% set current_location_path = location.path|slice(1) %}
```

Comment:
How will this function in multisite scenario? Does it perhaps need `|slice(2)` ?

Reply:
Nah, doesn't really matter, it will work both ways. We're just removing the tree root location ID (1) because it is not needed, to reduce the number if iterations in the for loop by one - because we know for sure we will never match any menu items with data-location-id="1". Micro-optimization, but we can just drop out the slice completely if you want.

Reply:
I would, since it could (obviously) lead to wrong conclusions.